### PR TITLE
relay-example: Add range add mutation example

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

**Comment by tbergq** [comment]
> Just a suggestion to add few words on top of the example page explaining what's this about.

Good idea, I will add it before we merge

**Comment by tbergq** [comment]
@michalsanger, looks ok?

**Comment by michalsanger** [comment]
@tbergq just not sure about the `Alert`. I used this markup:
https://github.com/adeira/universe/blob/master/src/example-relay/src/LocalForm/index.js#L78-L85

**Comment by tbergq** [comment]
Ok, I will use the same

**Comment by tbergq** [comment]
better @michalsanger ?

**Comment by michalsanger** [comment]
@tbergq the best :-)

File: src/example-relay/src/mutations/RangeAdd/LocationsForm.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  const [state, dispatch] = React.useReducer(reducer, initialState);
+  const [loading, setLoading] = React.useState(false);
+  const env = useRelayEnvironment();
+  const onSubmit = (e: SyntheticEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    if (state.type === '') {
+      alert('You must select a type');
+      return;
+    }
+    setLoading(true);
```
Comment (mrtnzlml) [inline src/example-relay/src/mutations/RangeAdd/LocationsForm.js]: You should use our `useMutation` hook instead. It returns `isPending` boolean so you don't have to deal with it manually like this.

**Pull request merged**